app/views/content/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.md

```diff
@@ -4,10 +4,10 @@ title: Update your subject and teaching knowledge
 meta_description: Find out how to update your subject and teaching knowledge to help return to teaching.
 date_posted: 17/02/2025
 category_tags: apply
-card-image: "/content-assets/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.jpg"
+card-image: "/content-assets/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.png"
 ---
 
-![A student working at a computer while a teacher helps.](/content-assets/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.jpg)
+![A student working at a computer while a teacher helps.](/content-assets/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.png)
```

Comment:
I think this is the only place where `/content-assets/jobseeker-guides/return-to-teaching-in-england/update-your-subject-and-teaching-knowledge.jpg` was used. Should the `jpg` asset be removed if not used anymore?